tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/options.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import PageRegion
+
+
+class SmsOptions(PageRegion):
+
+    _root_locator = (By.CSS_SELECTOR, 'form.visible')
+
+    def __init__(self, marionette):
+        root = marionette.find_element(*self._root_locator)
+        PageRegion.__init__(self, marionette, root)
+        self.wait_for_panel_to_appear()
+
+    def wait_for_panel_to_appear(self):
+        Wait(self.marionette).until(lambda m: self.root_element.rect['y'] == 0)
```

Comment:
Here too, I would prefer it if you would also check for root_element.is_displayed().